storage-resize-images/functions/src/index.ts

```diff
@@ -39,10 +39,6 @@ admin.initializeApp();
 
 logs.init();
 
-/**
- * When an image is uploaded in the Storage bucket We generate a resized image automatically using
- * ImageMagick which is installed by default on all Cloud Functions instances.
- */
```

Comment:
instead removing the comment, can you update it to reflect that it is using Sharp?